### PR TITLE
Verify the token are encrypted in kibana log

### PR DESCRIPTION
**Pull request by anpingli**
https://bugzilla.redhat.com/show_bug.cgi?id=1925081 
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/318450/console

### CODE REVIEW

**Comment by anpingli** [comment]
@QiaolingTang  @gkarager  Could you review this PR?

File: features/logging/kibana.feature [hunk 1]
```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+            | logging-infra=kibana |
+    When I run the :logs admin command with:
+      | resource_name | <%= pod.name %>                |
+    Then the output should not contain "XXXXXXXXXXXXXXXXXXXXXXXX"
```
Comment (QiaolingTang) [inline features/logging/kibana.feature:101]: Hi @anpingli , what is "XXXXXXXXXXXXXXXXXXXXXXXX"?

File: features/logging/kibana.feature [hunk 2]
```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+            | logging-infra=kibana |
+    When I run the :logs admin command with:
+      | resource_name | <%= pod.name %>                |
```
Comment (QiaolingTang) [inline features/logging/kibana.feature:100]: Too many spaces, and you should point out container name here.

File: features/logging/kibana.feature [hunk 3]
```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+            | logging-infra=kibana |
```
Comment (QiaolingTang) [inline features/logging/kibana.feature:98]: Too many spaces.

File: features/logging/kibana.feature (hunk 1 above)
Comment (anpingli) [reply features/logging/kibana.feature:101]: @QiaolingTang  That is the encrypted token.

Comment (anpingli) [reply features/logging/kibana.feature:101]: {"type":"response","@timestamp":"2021-03-29T09:38:29Z","tags":[],"pid":118,"method":"post","statusCode":200,"req":{"url":"/elasticsearch/_msearch?rest_total_hits_as_int=true&ignore_throttled=true","method":"post","headers":{"host":"kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.96 Safari/537.36","content-length":"955","accept":"application/json, text/plain, */*","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9","content-type":"application/x-ndjson","forwarded":"for=66.187.232.127;host=kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com;proto=https","kbn-version":"6.8.1","origin":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","referer":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com/app/kibana","sec-fetch-dest":"empty","sec-fetch-mode":"cors","sec-fetch-site":"same-origin","x-forwarded-access-token":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","x-forwarded-email":"XXXXXXXXXXXXXXXXXXXXXXXX","x-forwarded-for":"66.187.232.127, 10.129.2.21","x-forwarded-host":"kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","x-forwarded-port":"443","x-forwarded-proto":"https","x-forwarded-user":"testuser-0","securitytenant":"__user__"},"remoteAddress":"127.0.0.1","userAgent":"127.0.0.1","referer":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com/app/kibana"},"res":{"statusCode":200,"responseTime":75,"contentLength":9},"message":"POST /elasticsearch/_msearch?rest_total_hits_as_int=true&ignore_throttled=true 200 75ms - 9.0B"}

File: features/logging/kibana.feature [hunk 4]
```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+      | logging-infra=kibana |
+    When I run the :logs client command with:
+      | resource_name | <%= pod.name %> |
```
Comment (QiaolingTang) [inline features/logging/kibana.feature:100]: @anpingli I think we need point out the container name here as there are 2 containers in kibana pod, or the step will fail
```
When I run the :logs client command with:
      | resource_name | <%= pod.name %> |
      | c                        | xxxxxxxxxxxxxx       |
```

**Comment by anpingli** [comment]
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/318517/console

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**